packages/zero-pg/src/web.ts

```diff
@@ -64,10 +65,13 @@ export class PushProcessor<
 
   async process(
     mutators: MD,
-    queryString: unknown,
-    body: unknown,
+    queryString: URLSearchParams | Record<string, string>,
```

Comment:
I think it would be good to add some documentation too to this method (and generally any public entrypoints).

Reply:
(I mean comment-based docs that will show up in intellisense)